### ISSUE TITLE
Windows LLVM Versions (11.1.0, 12.0.1, 13.0.0)

### ISSUE THREAD

**Issue by gingerBill** [odin-lang/Odin#1246]
If you require a specific LLVM version on Windows for your builds (because of issues with LLVM), there are three branches which I recommend you merge (or patch) _against_:

LLVM 11.1.0: https://github.com/odin-lang/Odin/tree/windows-llvm-11.1.0
LLVM 12.0.1: https://github.com/odin-lang/Odin/tree/windows-llvm-12.0.1
LLVM 13.0.0: https://github.com/odin-lang/Odin/tree/windows-llvm-13.0.0

**WARNING**: These branches may be not be up to date with `master` but as they only update the LLVM-C headers and binaries, it should be safe to merge those branches against your own branch.

**Comment by Feoramund** [comment]
Are any of these versions supported any more?

**Comment by Kelimion** [comment]
No. Practically speaking the only version we support on Windows is the one reported by the LLVM-C.dll we ship with the compiler. It may compile against different LLVM versions as accepted by `build*`, but that's not really relevant on Windows.
